Scripts/demand/trips.py

```diff
@@ -90,12 +89,14 @@ def create_population(self):
         list
             Person
         """
+        numpy.random.seed(param.population_draw)
```

Comment:
The other random seed is set in `ModelSystem`. Maybe we could put them in the same place?

Reply:
This needs to be set here because `numpy` module (unlike `random`) inits random seed when module is imported. 

Moving to Python 3 helps as then we can use `random.choices` for weighted random choices.